backend/app/serializers/spree/admin/reports/top_products_by_line_item_totals/csv_serializer.rb

```diff
@@ -0,0 +1,21 @@
+require 'csv'
+
+module Spree
+  module Admin
+    module Reports
+      module TopProductsByUnitSold
+        class CsvSerializer
+          def call(objects)
+            CSV.generate do |csv|
+              csv << %w[sku line_item_totals]
+
+              objects.each do |sku, line_item_totals|
```

Comment:
@mbajur wdyt about change `each` to `find_each` ?

Reply:
`objects` is an array, not ActiveRecord collection so we can't use it in here.